### DISCUSSION TITLE
Terminal hyperlink opens system default app instead of configured action

### DISCUSSION THREAD

**Discussion by bernardassan** [kovidgoyal/kitty#3977] [Q&A] [answered]
I am trying out [scripting the mouse](https://sw.kovidgoyal.net/kitty/open_actions/#scripting-the-mouse-clickl) . After pasting the sample code on the page in my config and reloading config. I issue` ls --hyperlink=auto` and hold `ctrl+shif` and click the name of the image but it opens with system default .ie eog instead of the icat kitten and same for the log file example it opens with libreoffice instead of tailing in a new window

**Comment by kovidgoyal** [comment]
On Sun, Aug 29, 2021 at 01:48:49AM -0700, Malpha wrote:
> I am trying out [scripting the mouse](https://sw.kovidgoyal.net/kitty/open_actions/#scripting-the-mouse-clickl) . After pasting the sample code on the page in my config and reloading config. I issue` ls --hyperlink=auto` and hold `ctrl+shif` and click the name of the image but it opens with system default .ie eog instead of the icat kitten and same for the log file example it opens with libreoffice instead of tailing in a new window

Make sure you are running a recent version of kitty. Check the changelog
for when the feature was introduced.

**Reply by bernardassan** [reply]
I am using kitty version 0.23.1

**Comment by kovidgoyal** [comment]
You paste it in ~/.config/kitty/open-actions.conf and then you restart kitty, reloading the config only realods kitty.conf not open-actions.conf

**Reply by bernardassan** [reply]
So then how to I reload kitty so that it reloads all my config files

**Comment by kovidgoyal** [comment, marked as answer]
*restart* kitty. AKA quit it and start it again.


On Sun, Aug 29, 2021 at 04:18:38PM -0700, Malpha wrote:
> So then how to I reload kitty so that it reloads all my config files
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/3977#discussioncomment-1251409

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by bernardassan** [reply]
Okay that work Thanks a lot. But I think it will be better if reloading .ie `ctrl+shift+f5` loads all the config files in ~/.config/kitty because for my case I have session startup file with a lot of tabs and windows ,so having to close all and run kitty again only for some config files to reload isn't ideal one bit. Because I usually never close my kitty instance after first run , if I need a new kitty instance I just create a new system window or run a new kitty instance from my current session and maybe in the background if need be. It will be nice to get reloading to doing that

**Reply by bernardassan** [reply]
I think I see some work going in that direction. Thanks a lot . I really appreciate it

**Reply by kovidgoyal** [reply]
Its already implemented in master and you are welcome.
